packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs

```diff
@@ -13,19 +13,12 @@ namespace UnbrandedTypeSpec
     /// <summary></summary>
     public partial class UnbrandedTypeSpecClient
     {
-        private ClientPipeline _pipeline;
```

Comment:
I would probably leave this on the client since create request methods won't use it at all I don't think

Reply:
Create request method uses the pipeline: https://github.com/Azure/autorest.csharp/blob/feature/v3/test/UnbrandedProjects/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs#L1276

Reply:
Discussed offline: We will move things to the rest client file that are ONLY used in the rest client. If its used in both we would leave it on the main client file.